### PR TITLE
Added validation examples.

### PR DESCRIPTION
**Pull request by andresuribe87**
This aims to address #143 for VC claimsets (i.e. a VC without the proof piece being referenced.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/pull/195.html" title="Last updated on Aug 7, 2023, 10:07 PM UTC (6062d4e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/195/6aae47c...6062d4e.html" title="Last updated on Aug 7, 2023, 10:07 PM UTC (6062d4e)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -564,6 +564,118 @@ <h3>Evaluation</h3>
...
+            "credentialSubject": {
+              "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+              "emailAddress": "subject@example.com"
+            },
+            "credentialSchema": {
+              "id": "https://example.com/schemas/email.json",
+              "type": "JsonSchema",
+            }
+          }
+        </pre>
```
Comment (decentralgabe) [inline index.html:612]:
```suggestion
         </pre>
```

**Review by decentralgabe — COMMENTED** [review]
I think this is good to add. I would additionally link to the validation spec draft that would be used/followed for each example.

One thing I think we should do to make this easier to view is having a tabbed example viewer like this https://www.w3.org/TR/vc-data-model/#example-usage-of-the-type-property, you can see the respec code for it here https://github.com/w3c/vc-data-model/blob/main/REC/2022-03-03/index.html#L2525

**Comment by decentralgabe** [comment]
Also I would say #143 would not be fully addressed until we show examples with at least data integrity and having it packaged as a JWT

**Comment by andresuribe87** [comment]
@decentralgabe PTAL, should have addressed all the feedback.

**Review by decentralgabe — APPROVED** [review]

**Pull request merged**